tyr-core/src/main/java/org/jboss/tyr/github/GitHubService.java

```diff
@@ -49,6 +49,10 @@
     @Inject
     TyrConfiguration configuration;
 
+    public boolean commitCheckDisabled() {
```

Comment:
is this wrapper needed? Can't you access configuration when you need it?

Reply:
Unfortunately I wasn't able to access the configuration differently without a need of extensive changes to the existing SkipCheck Tests. Also don't wanted to inject TemplateChecker, from where I could access the required method. Because of that I've made this small workaround wrapper.

Reply:
Implemented fix based on recommendation.